app/models/nearby_facility.rb

```diff
@@ -72,9 +72,7 @@ def get_facilities_in_isochrone(params, isochrone_response)
         # find all facilities that lie inside of the polygon
         conditions = "ST_Intersects(#{make_polygon}, ST_MakePoint(long, lat))"
         facilities_query_base_instance = FacilitiesQuery::Base.new(params)
-        BaseFacility::TYPES.flat_map do |facility_type|
-          facilities_query_base_instance.get_facility_data(conditions, params[:type], facility_type, params[:services])
-        end
+        Array.new facilities_query_base_instance.get_facility_data(conditions, params[:type], params[:type], params[:services])
```

Comment:
I think you can do a `.to_a` at the end of the line here instead of using Array.new. Not sure

Reply:
Although, why are we making this an array instead of a relation? Does it matter?